README.md

```diff
@@ -126,6 +126,49 @@ Try the following examples to learn more about Dapr's Java SDK:
 
 Please, refer to our [Javadoc](https://dapr.github.io/java-sdk/) website.
 
+#### Reactor API
+
+The Java SDK for Dapr is built using [Project Reactor](https://projectreactor.io/). It provides an asynchronous API for Java. When consuming a result is consumed synchronously, as in the examples above, the `block()` method is used.
+
+The code below does not make any API call, it simply returns the [Mono](https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html) publisher object:
+
+```java
+daprClient.publishEvent("mytopic", "my message");
```

Comment:
May want to show this assigned to Mono.  And it can be combined with the next line by calling block on the returned mono.  eg roughly

"This one does not start execution":
Mono<Void> m = daprClient.publishEvent("mytopic", "my message");

"To start execution and receive the result call subscribe/block/etc on it."